Resources/public/js/views/serverside/ez-roleserversideview.js

```diff
@@ -41,10 +44,10 @@ YUI.add('ez-roleserversideview', function (Y) {
          */
         _pickSubtree: function (e) {
             var button = e.target,
-                unsetLoading = Y.bind(this._uiUnsetAssignRoleLoading, this, button);
+                unsetLoading = Y.bind(this._uiUnsetUDWButtonLoading, this, button);
```

Comment:
@yannickroger It **is** called what it does: unset the loading state of the UDW button. To clarify, the new usage comes in addition to the old one - it is used both when assigning roles, and when editing policy limitations. But what it **does** is the same in both cases. So I named it after what it does :)